### PR TITLE
Analysis annotations

### PR DESCRIPTION
**Pull request by kdaily**
Opening a PR for @blogsdon's issue #4 so we can comment/review specific regions.

### CODE REVIEW

File: minimal_Sage_analysis.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  - principalComponents
+  - genomeWideAssociation
+  - clustering
+  - supervisedLearning
+  - batchEffectCorrection
+  - anova
+  - variancePartition
+  - moduleIdentification
+
+analysisMethod:
```
Comment (kdaily) [inline minimal_Sage_analysis.yml]: Merge this into above `analysisType`

File: minimal_Sage_analysis.yml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  - networkReconstruction
+  - clustering
+  - batchEffectAdjustment
+  - supervisedLearning
+  - multipleTestingAdjustment
+  - regression
+
+networkReconstructionMethod:
```
Comment (kdaily) [inline minimal_Sage_analysis.yml]: rename to `statisticalNetworkReconstructionMethod`

File: minimal_Sage_analysis.yml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  - c3net
+  - MEGENA
+
+clusteringMethod:
+  - hierarchicalClustering
+  - kMeans
+  - WGCNA
+  - principalComponentAnalysis
+
+batchEffectAdjustmentMethod:
```
Comment (kdaily) [inline minimal_Sage_analysis.yml]: Should be `batchEffectCorrectionMethod`?

File: minimal_Sage_analysis.yml [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+  - barChart
+
+validationMethod:
+  - sequenceValidation
+  - targetValidation
+  - CRISPR-CAS9
+  - RNAi
+  - shRNA
+
+alignmentMethod:
```
Comment (kdaily) [inline minimal_Sage_analysis.yml]: Maybe should be in minified or project-specific standard?

File: minimal_Sage_analysis.yml (hunk 1 above)
Comment (blogsdon) [inline minimal_Sage_analysis.yml]: fixed

File: minimal_Sage_analysis.yml (hunk 2 above)
Comment (blogsdon) [inline minimal_Sage_analysis.yml]: fixed

File: minimal_Sage_analysis.yml (hunk 3 above)
Comment (blogsdon) [inline minimal_Sage_analysis.yml]: fixed

File: minimal_Sage_analysis.yml (hunk 4 above)
Comment (blogsdon) [inline minimal_Sage_analysis.yml]: won't fix

**Pull request merged**